spec/validation/meta_yml_spec.rb

```diff
@@ -1,103 +1,140 @@
 # frozen_string_literal: true
 require "spec_helper"
+require "pry"
 
-module HathiTrust
-  module Validation
+include HathiTrust
```

Comment:
Reopening the module seems superior to this.  I'm not even sure what an include actually does here.